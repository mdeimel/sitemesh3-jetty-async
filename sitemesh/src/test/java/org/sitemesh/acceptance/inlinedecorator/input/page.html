<html>
  <head>
    <title>Some Example</title>
  </head>
  <body>

    <sitemesh:decorate decorator="/inline-decorator.html" title="Inline 1">
      <p>Some inline decorated content.</p>
    </sitemesh:decorate>

    <sitemesh:decorate decorator="/inline-decorator.html" title="Inline 2">
      <p>More inline decorated content.</p>
    </sitemesh:decorate>

    <p>More content.</p>

  </body>
</html>