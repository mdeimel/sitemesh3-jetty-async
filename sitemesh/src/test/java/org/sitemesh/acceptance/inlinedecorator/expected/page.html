<!-- Common header -->
<html>
  <head>
    <title>Some Example</title>
    <style type="text/css">
      /* common css */
      body { padding: 20px; }
      .title { color: #00c; }
    </style>
  </head>
  <body>

    <h2 class="title">
      Page: Some Example
    </h2>

    <!-- body -->
    <div class="inline">
      <h2 class="title">
        Inline 1
      </h2>
      <p>Some inline decorated content.</p>
    </div>

    <div class="inline">
      <h2 class="title">
        Inline 2
      </h2>
      <p>More inline decorated content.</p>
    </div>

    <p>More content.</p>

    <!-- end of body -->

  </body>
</html>
<!-- Common footer -->